cloudinit/config/cc_ansible.py

```diff
@@ -196,3 +211,15 @@ def run_ansible_pull(pull: AnsiblePull, cfg: dict):
     )
     if stdout:
         sys.stdout.write(f"{stdout}")
+
+
+def ansible_galaxy(cfg: dict, ansible: AnsiblePull):
+    actions = cfg.get("actions", [])
+    ansible_config = cfg.get(CFG_OVERRIDE, "")
```

Comment:
cfg here refers to galaxy, right? Galaxy doesn't have an 'ansible_config' key under it, so I don't think this will work.